src/Common/ErrorCodes.cpp

```diff
@@ -681,6 +681,23 @@ namespace ErrorCodes
 
     ErrorCode end() { return END + 1; }
 
+    size_t increment(
```

Comment:
Let's avoid introducing this new code here (and in Exception), and simply obtain `query_id` from the TLS - `CurrentThread::getQueryId()`